core/src/main/java/hudson/tasks/ArtifactArchiver.java

```diff
@@ -113,7 +113,7 @@
     /**
      * Indicate whether symbolic links should be followed or not
      */
-    @Nonnull
+    @NonNull
```

Comment:
then surely this should be a `boolean` :-o  (not a comment on the PR this a WTF comment)

Reply:
🤦‍♂